cartridges/int_stickyio_sfra/cartridge/controllers/Checkout.js

```diff
@@ -6,42 +6,69 @@ server.extend(Checkout);
 
 var stickyioEnabled = require('dw/system/Site').getCurrent().getCustomPreferenceValue('stickyioEnabled');
 var stickyioForceRegisteredCheckout = require('dw/system/Site').getCurrent().getCustomPreferenceValue('stickyioForceRegisteredCheckout');
+var sfccVersion60 = require('dw/system/Site').getCurrent().getCustomPreferenceValue('stickyioSFCCVersion60');
 
 if (stickyioEnabled) {
     var stickyio = require('~/cartridge/scripts/stickyio');
-
+    if (!sfccVersion60) {
+	    server.append(
+	        'Login',
+	        function (req, res, next) {
+	            var checkoutView = res.getViewData();
+	            checkoutView.allowGuestCheckout = true;
+	            if (stickyio.hasSubscriptionProducts() !== false) {
+	                checkoutView.stickyioForceRegisteredCheckout = stickyioForceRegisteredCheckout;
+	                checkoutView.stickyioOrder = true;
+	            }
+	            res.setViewData(checkoutView);
+	
+	            return next();
+	        }
+	    );
+	    
+    }
+    
     server.append(
-        'Login',
+        'Begin',
         function (req, res, next) {
+            var URLUtils = require('dw/web/URLUtils');
+            if (!sfccVersion60 && stickyioForceRegisteredCheckout && !req.currentCustomer.profile) { res.redirect(URLUtils.url('Checkout-Login')); }
+
+			
             var checkoutView = res.getViewData();
+            checkoutView.sfccVersion60 = sfccVersion60;
             if (stickyio.hasSubscriptionProducts() !== false) {
+            	if (stickyioForceRegisteredCheckout) {
+            		checkoutView.allowGuestCheckout = false;
+            	} else {
+            		checkoutView.allowGuestCheckout = true;
+            	}
```

Comment:
Please correct spacing issues in lines 41-45?

Reply:
fixed